### PR TITLE
Prepare for 0.9.0.0 release

### PR DESCRIPTION
**Pull request by alanz**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,44 @@
+# 0.9.0.0
+
+- GHC 8.6.5 preliminary support added via the nightly build (@alanz)
+- Add `diagnosticsOnChange` config parameter, default `True`
+  (preserving prior hie behaviour). Setting it `False` only generates
+  diagnostics on file save. (#1164, @mpickering/@lorenzo)
```
Comment (Infinisil) [inline Changelog.md]: It would be nice if these PR numbers were clickable. A find-and-replace incantation should do it.

**Comment by alanz** [comment]
Damn, I thought github was clever enough to do that. That was my intention.

**Comment by alanz** [comment]
@bubba please tag and release this when CI passes, I will be spottily available the next 2 days

**Comment by nponeccop** [comment]
Snapshot bumping line is missed. We have already bumped 8.6.4 and may bump 8.6.3 due to #1236

**Comment by nponeccop** [comment]
Older versions have it in the following format:

`Resolver bumped, LTS 13.10 for GHC 8.6.3, LTS 13.15 for GHC 8.6.4 (@alanz)`

In our case it's

`Resolver bumped, LTS 13.19 for GHC 8.6.4 (@alanz)`

**Pull request merged**